main.go

```diff
@@ -18,18 +39,69 @@ func getPing(w http.ResponseWriter, r *http.Request) {
 
 }
 
-func getBooks(w http.ResponseWriter, r *http.Request) {
+func Books(w http.ResponseWriter, r *http.Request) {
+	metodo := r.Method
+	fmt.Printf("metodo: %v\n", metodo)
+	switch {
+	case metodo == "GET":
+		getBooks(w, r)
+	case metodo == "POST":
+		postBooks(w, r)
+	default:
+		w.WriteHeader(http.StatusMethodNotAllowed)
+	}
+}
 
-	bookslist := []struct { //Struct created to handle data of the list of books
-		Name      string  `json:"name"`
-		Price     float32 `json:"price"`
-		Inventory int     `json:"inventory"`
-	}{
-		{"Book 1", 30.20, 2},
-		{"Book 2", 20.30, 1},
-		{"Book 3", 32.20, 5},
+func postBooks(w http.ResponseWriter, r *http.Request) {
+	//TO DO:
```

Comment:
Todo?

Reply:
Ah, yes. Before start coding, I've made a list:
//TO DO:
//Read the Json body
//Verify if the entry is in a valid format
//Verify if that book already exists in the database
//Atribute an ID to the entry
//Store the book in the database	
//Return a sucess message

I'll have to clean better.